frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -10,18 +10,16 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
-  let title = odhApp.spec.displayName;
+  const title = odhApp.spec.displayName;
   let icon;
```

Comment:
I would even convert the icon into a constant and then conditionally render it (checking `isRedHatSupported(odhApp)`) in the JSX part. But it seems a great solution, this was a really werid hack.